torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -346,16 +346,23 @@ void ScriptModuleSerializer::serialize(
   // so loading the code does not depend on loading the data
   std::vector<IValue> ivalue_constants(
       constant_table_.begin(), constant_table_.end());
-  writeArchive(
-      c10::ivalue::Tuple::create(ivalue_constants),
-      /*archive_name=*/kArchiveNameConstants,
-      /*archive_dir=*/"",
-      /*tensor_dir=*/"constants/");
   if (bytecode_format) {
+    writeArchive(
+        c10::ivalue::Tuple::create(ivalue_constants),
+        /*archive_name=*/kArchiveNameConstants,
+        /*archive_dir=*/"",
+        /*tensor_dir=*/"constants/",
+        true);
```

Comment:
Add comment on this arg, like the args above.

Reply:
Add it in the latest commit.